README.md

```diff
@@ -1,54 +1,201 @@
+[![GitHub latest release](https://img.shields.io/github/release/TASVideos/BizHawk.svg?logo=github&logoColor=333333&style=popout)](https://github.com/TASVideos/BizHawk/releases/latest)
+[![dev builds | AppVeyor](https://img.shields.io/badge/dev_builds-AppVeyor-orange.svg?logo=appveyor&logoColor=333333&style=popout)](https://ci.appveyor.com/project/zeromus/bizhawk-udexo/history)
+[![Windows prereqs | GitHub](https://img.shields.io/badge/Windows_prereqs-GitHub-darkred.svg?logo=github&logoColor=333333&style=popout)](https://github.com/TASVideos/BizHawk-Prereqs/releases/latest)
 
-[Release Notes](http://tasvideos.org/Bizhawk/ReleaseHistory.html)
+A multi-system emulator written in C#. As well as quality-of-life features for casual players, it also has recording/playback and debugging tools, making it the first choice for TASers (Tool-Assisted Speedrunners).
 
-[Here](http://tasvideos.org/Bizhawk/Features.html) is a list of features offered by bizhawk.
+Click the "release" button above to grab the latest stable version ([changelog at TASVideos](http://tasvideos.org/Bizhawk/ReleaseHistory.html)). 
 
-## Download Binaries
+New users on Windows should click the "prereqs" button too, see *Installing* for info.
 
-Windows users, don't forget to run the [prereq installer](http://github.com/TASVideos/BizHawk-Prereqs/releases) first!
+**Never mix different versions** of BizHawk — Keep each version in its own folder.
 
-Release binaries can be found on [on github](http://github.com/TASVideos/BizHawk/releases)
+Jump to:
+* [Installing](https://github.com/TASVideos/BizHawk/blob/master/README.md#installing)
+	* [Windows 7/8.1/10](https://github.com/TASVideos/BizHawk/blob/master/README.md#windows-78110)
+	* [GNU+Linux and macOS](https://github.com/TASVideos/BizHawk/blob/master/README.md#gnulinux-and-macos)
+* [Building](https://github.com/TASVideos/BizHawk/blob/master/README.md#building)
+* [Usage](https://github.com/TASVideos/BizHawk/blob/master/README.md#usage)
+	* [TASing](https://github.com/TASVideos/BizHawk/blob/master/README.md#tasing)
+	* [Cores](https://github.com/TASVideos/BizHawk/blob/master/README.md#cores)
+* [Support and troubleshooting](https://github.com/TASVideos/BizHawk/blob/master/README.md#support-and-troubleshooting)
+* [Contributing](https://github.com/TASVideos/BizHawk/blob/master/README.md#contributing)
+	* [Testing](https://github.com/TASVideos/BizHawk/blob/master/README.md#testing)
+* [Related projects](https://github.com/TASVideos/BizHawk/blob/master/README.md#related-projects)
+* [License](https://github.com/TASVideos/BizHawk/blob/master/README.md#license)
 
-[Developer build](https://ci.appveyor.com/project/zeromus/bizhawk-udexo/build/artifacts) of the most recent commit
+## Features and systems
 
-Never mix different versions of BizHawk. Keep each version in its own folder.
+The BizHawk common features (across all cores) are:
+* format and region detection for game images
+* image corruption warning checked against database
```

Comment:
These 2 can be combined. "format, region, and integrity detection for game images"? Actually it's not about corruption or integrity, just about recognizing something as GOOD [!] image. This means an image matches the original/official game.

Reply:
for some reason I unlearned the word "integrity"! that's much nicer